inventory_management_system_api/repositories/catalogue_item.py

```diff
@@ -107,5 +113,18 @@ def list(self, catalogue_category_id: Optional[str]) -> List[CatalogueItemOut]:
             logger.info("%s matching the provided catalogue category ID filter", message)
             logger.debug("Provided catalogue category ID filter: %s", catalogue_category_id)
 
-        catalogue_items = self._collection.find(query)
+        catalogue_items = self._catalogue_items_collection.find(query)
         return [CatalogueItemOut(**catalogue_item) for catalogue_item in catalogue_items]
+
+    def has_child_elements(self, catalogue_item_id: CustomObjectId) -> bool:
+        """
+        Check if a catalogue item has children elements based on its ID.
+
+        Children elements in this case means whether or not a catalogue item has children items
+
+        :param catalogue_item_id: The ID of the catalogue item to check
+        :return: True if the catalogue item has children elements, False otherwise.
+        """
+        logger.info("Checking if catalogue item with ID '%s' has children elements", catalogue_item_id)
+        item = self._items_collection.find_one({"catalogue_item_id": catalogue_item_id})
+        return item
```

Comment:
```suggestion
        return item is not None
```
This is returning the item, not a boolean.